sdl-core/src/test/scala/io/smartdatalake/testutils/TestUtil.scala

```diff
@@ -158,7 +158,7 @@ object TestUtil extends SmartDataLakeLogger {
           .port(port)
           .httpsPort(httpsPort)
           .bindAddress(host)
-          .keystorePath("src/test/resources/test_keystore.pkcs12")
+          .keystorePath("sdl-core/src/test/resources/test_keystore.pkcs12")
```

Comment:
@zzeekk I think this addition is the reason for the failed unit test:
`Cause: java.io.FileNotFoundException: /home/runner/work/smart-data-lake/smart-data-lake/sdl-core/sdl-core/src/test/resources/test_keystore.pkcs12 (No such file or directory)`